### PR TITLE
[Doc Issue][Stop distribution of an app][2329212]

### PR DESCRIPTION
**Pull request by v-dreddipogu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2103f1bfc25bc2710be5b198e4459a449cf0daaf/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fpost-publish%2Foverview.md) | 87 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f5c11c8a-fc2c-4805-99d2-3411bc424c38) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/82fb96e156d40494cdb9cffa3fbf3c7db8e9bfa6/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fpost-publish%2Foverview.md) | 87 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2e07668d-5512-40f0-a08c-0b89558c05b8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/658a4422dbf88c6bae036bb9bcd79415c18ea636/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fpost-publish%2Foverview.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/da115c2a-46b6-47ad-8688-4461883c3eb3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/61e17737f60b1e9868c3e865a10f0c38bf988778/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fpost-publish%2Foverview.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/caed6265-c767-47d0-b154-a1c46bf3d610) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/77903324cb247cc6dd5378333395ee5b0b0f73d2/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fpost-publish%2Foverview.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9681f1ad-e6eb-4321-95f2-c8c1cdd4be8e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/200cfcb0425b9a8124f41ba5e8c8443eeba55c0e/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fpost-publish%2Foverview.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2a892ef2-cbc5-4c12-8853-d647393cd527) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/068bd57f1a65c3a36e78a792816a7f396fc16626/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fpost-publish%2Foverview.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/98b6af28-effa-4376-afac-c1d7426a3f70) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/dbad293d12bc5877ae6a9650842799d827e10bb4/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fpost-publish%2Foverview.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/33aff005-fb9e-4137-8c18-dc89706d5bdb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5c43c3aec81a7a4a97edd9874c00d331a7070b0e/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fpost-publish%2Foverview.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4b204f27-4f1e-45e4-a516-46ebaeae3899) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md [hunk 1]
```diff
@@ -60,6 +60,19 @@ Create the link using the following URL appended with your app ID: `https://team
 
 [Microsoft 365 Certification](/microsoft-365-app-certification/docs/certification) offers assurances that data and privacy are adequately secured and protected when a third-party Office app or add-in is installed in your Microsoft 365 ecosystem. Certification confirms that your app is compatible with Microsoft technologies, compliant with cloud app security best practices, and supported by Microsoft.
 
+## Stop app distribution
+
+You can remove apps from the [Microsoft commercial marketplace](/azure/marketplace/overview), which prevents new customers from finding and using them.
+
+**To stop distribution of an app after you've published, follow the steps:**
```
Comment (ashishguptaiitb) [inline msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md:67]: No need to emphasize this. We discussed that this when talking about markdownlint reported warnings.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c679995b1c4a034bdafad10055bff069527cc1d7/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fpost-publish%2Foverview.md) | 88 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/777381e3-fdfb-4c8e-b465-a6dd818f6235) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/158e68bf7b4865efe4a841eeea5968e0589e2187/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fpost-publish%2Foverview.md) | 88 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bcacbd19-e94d-4550-b242-4161b77b7de9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5821a0a25f001adaa0e561f17461299698d779ac/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fpost-publish%2Foverview.md) | 88 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/74a55153-91c2-4784-814a-4749eb7ebcdc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3908bd067bf73499c8b6a7251f9ef7f29ff31557/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fpost-publish%2Foverview.md) | 88 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b93ac775-4237-4c1c-994e-e7b4848de78f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e3d4c2dfbfc650c8f09a1cca97d768d654406c59/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fpost-publish%2Foverview.md) | 88 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bdf040e2-adde-4463-8329-95fe2810f8a6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9dd103cdff9e730bc9c9dfa79a2d10bcf3839f14/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fpost-publish%2Foverview.md) | 88 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/08830174-cb6c-4a4a-b7c0-ec09aceba64e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0315c5cae7dd8c6e5ef505ddb3772c60898dc3d3/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fpost-publish%2Foverview.md) | 88 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/302bdf07-473c-4c07-a42e-b33ada3c40c6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**